cmd/yggctl/actions.go

```diff
@@ -185,6 +186,12 @@ func workersAction(c *cli.Context) error {
 		return cli.Exit(fmt.Errorf("cannot list workers: %v", err), 1)
 	}
 
```

Comment:
I would add here some comment explaining, why we delete such workers. At the first glance, it looks strange that we do something like this. It has some historical reason and it should be mentioned here in comment.

Reply:
Thank you @jirihnidek and @subpop for the feedback.

I had come from the Ruby world where comments were highly frowned upon. But since I'm an old `C` programmer I surely can do comments! 😄 and I will update the commit message.

Reply:
@jirihnidek and @subpop 

I believe this has been resolved by commit: https://github.com/RedHatInsights/yggdrasil/pull/272/commits/152af66a40e2c4f846fb69be4b37c139902acc23

Please take a look.

Thank you again for the help.